bundles/org.openhab.binding.meater/src/main/feature/feature.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.meater-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-binding-meater" description="meater Binding" version="${project.version}">
```

Comment:
Minor but "Meater Binding" will be better and coherent with other bindings.

Reply:
Fixed!